app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -100,7 +105,7 @@ module.exports = React.createClass
       @handleFocus()
 
   handleFocus: ->
-    @refs['select-0'].getInputNode().focus()
+    @refs['select-0'].focus()
```

Comment:
I'm trying to reproduce this but can't. The dropdown control is being focussed for both questions.